tools/dm/shard-merge.md

```diff
@@ -46,29 +46,29 @@ In the above example, the merging process is simplified, where only two MySQL in
+4. At `t3`, the sharding DDL events from instance 2 are received.
+5. From `t4` on, the Syncer unit receives the DML events of `schema V2` from instance 2 as well.
 
-Assume that we do not handle the DDL statements of sharded tables during the replication process. After DDL statements of the instance 1 are replicated to the downstream, the downstream table schema will be modified to `schema V2`. But for the instance 2, the Syncer unit in DM-worker is still receiving DML events of `schema V1` between `t2` to `t3`. Therefore, when the DML statements of `schema V1` are replicated to the downstream, the inconsistency between the DML statements and the table schema can cause errors and the data cannot be replicated successfully.
+Assume that the DDL statements of sharded tables are not processed during the replication process. After DDL statements of instance 1 are replicated to the downstream, the downstream table schema is changed to `schema V2`. But for instance 2, the Syncer unit in DM-worker is still receiving DML events of `schema V1` from `t2` to `t3`. Therefore, when the DML statements of `schema V1` are replicated to the downstream, the inconsistency between the DML statements and the table schema can cause errors and the data cannot be replicated successfully.
 
 ### Principles
 
-This section shows how DM replicates DDL statements in the process of merging sharded tables by continuing the above example in the [background](#background) section.
+This section shows how DM replicates DDL statements in the process of merging sharded tables based on the above example in the [background](#background) section.
 
 ![shard-ddl-flow](../../media/shard-ddl-flow.png)
 
 In this example, `DM-worker-1` replicates the data from MySQL instance 1 and `DM-worker-2` replicates the data from MySQL instance 2. `DM-master` coordinates the DDL replication among multiple DM-workers. Starting from `DM-worker-1` receiving the DDL statements, the DDL replication process is simplified as follows:
 
-1. `DM-worker-1` receives the DDL statement from MySQL instance 1 at `t1`, pauses the data replication of the corresponding DDL and DML statements itself, and sends the DDL information to `DM-master`.
-2. `DM-master` decides that the replication of this DDL statement needs to be coordinated based on the received DDL information, creates a lock for this DDL statement, sends the DDL lock information back to `DM-worker-1` and marks `DM-worker-1` as the owner of this lock at the same time.
+1. `DM-worker-1` receives the DDL statement from MySQL instance 1 at `t1`, pauses the data replication of the corresponding DDL and DML statements, and sends the DDL information to `DM-master`.
+2. `DM-master` decides whether to coordinate the replication of this DDL statement based on the received DDL information, creates a lock for this DDL statement, sends the DDL lock information back to `DM-worker-1` and marks `DM-worker-1` as the owner of this lock at the same time.
```

Comment:
It seems that using `whether` in this example is inappropriate. Same for other similar "whether" below.

Reply:
Agree. DM-master always coordinates the replication if needed.

Reply:
@CaitinChen So please update. Thanks!